core-httpclient-impl/gradle.properties

```diff
@@ -1 +0,0 @@
-httpClientVersion = 4.5.13
```

Comment:
Just checking (perhaps automatic):  Did you intend to remove (vs upgrade) this entry?  Can we delete the file if it's going to be empty?